app/views/appointments/new.html.erb

```diff
@@ -153,6 +153,12 @@
         <%= f.button class: 'btn btn-primary btn-block t-submit' do %>
           Make appointment and notify<br><%= @appointment_form.name %>
         <% end %>
+        <%= link_to 'Hide this Booking Request',
+          @appointment_form.location_aware_booking_request,
+          method: :delete,
+          class: 'btn btn-danger btn-block t-deactivate',
+          data: { confirm: 'Are you sure?' }
```

Comment:
Shall we make this a bit more contextual? 

`Are you sure you want to hide #{@appointment_form.name}'s booking request?`

Or something?